pyqt-apps/siriushla/common/afc_acq_core/triggers.py

```diff
@@ -33,7 +35,9 @@ def setupui(self):
 
     def get_trigger_groupbox(self, idx):
         trig = 'TRIGGER{0:d}'.format(idx)
-        name = trig + (': ' + self.names[idx] if self.names else '')
+        hltrig = 'Monit' if idx == 7 else HLTimeSearch.get_hl_from_ll_triggers(
+            self.afctiming.substitute(propty_name=f'CRT{idx}'))
```

Comment:
Is there a TODO item for a high level trigger for Monit?

Reply:
Yes @ericonr ! I talked offline to @fernandohds564 and we plan to implement two new high level triggers: SI-Fam:TI-BPM-MonitClk (AMC7) and SI-Fam:TI-BPM-OrbIntlk (AMC4)